crates/partition-store/src/invocation_status_table/mod.rs

```diff
@@ -169,10 +128,42 @@ fn get_invocation_status<S: StorageAccess>(
         })
 }
 
+fn try_migrate_and_get_invocation_status<S: StorageAccess>(
+    storage: &mut S,
+    invocation_id: &InvocationId,
+) -> Result<InvocationStatus> {
+    let _x = RocksDbPerfGuard::new("try-migrate-and-get-invocation-status");
+
+    let v1_key = create_invocation_status_key_v1(invocation_id);
+    if let Some(status_v1) = storage.get_value::<_, InvocationStatusV1>(v1_key.clone())? {
+        trace!("Migrating invocation {invocation_id} from InvocationStatus V1 to V2");
```

Comment:
I still think that adding the `V2` suffix makes sense as this log line shows. But what's more important is to be consistent.